app/experimenter/nimbus-ui/src/components/PageRequestReview/index.stories.tsx

```diff
@@ -53,267 +44,73 @@ storiesOf("pages/RequestReview", module)
-        }}
-      />
-    ),
-  )
-  .add(
-    "review approved in experimenter, not yet approved in remote settings, user does not have reviewer role",
-    () => (
-      <SubjectEXP1055
-        {...{
-          isLaunchRequested: true,
-          isLaunchApproved: true,
-          canReview: false,
-        }}
-      />
-    ),
-  )
-  .add(
-    "review rejected in experimenter or remote settings, user has reviewer role",
-    () => (
-      <SubjectEXP1055
-        {...{
-          isLaunchRequested: true,
-          isLaunchApproved: false,
-          canReview: true,
-          launchRequestedByUsername: "abc@mozilla.com",
-          rejectFeedback: mockRejectFeedback,
-        }}
-      />
-    ),
-  )
-  .add(
-    "review rejected in experimenter or remote settings, user does not have reviewer role",
-    () => (
-      <SubjectEXP1055
-        {...{
-          isLaunchRequested: true,
-          isLaunchApproved: false,
-          canReview: false,
-          launchRequestedByUsername: "abc@mozilla.com",
-          rejectFeedback: mockRejectFeedback,
-        }}
-      />
-    ),
-  )
-  .add(
-    "review approved in experimenter but request timed out in Remote Settings, user has reviewer role",
-    () => (
-      <SubjectEXP1055
-        {...{
-          isLaunchRequested: true,
-          canReview: true,
-          rsRequestTimedOut: true,
-          launchRequestedByUsername: "abc@mozilla.com",
-        }}
-      />
-    ),
-  )
-  .add(
-    "review approved in experimenter but request timed out in Remote Settings, user does not have reviewer role",
-    () => (
-      <SubjectEXP1055
-        {...{
-          isLaunchRequested: true,
-          canReview: false,
-          rsRequestTimedOut: true,
-          launchRequestedByUsername: "abc@mozilla.com",
-        }}
-      />
-    ),
-  );
-
-const SubjectDraftStatusOperationsWithActions = ({
-  rejectExperimentLaunch = action("rejectExperimentLaunch"),
-  approveExperimentLaunch = action("approveExperimentLaunch"),
-  confirmExperimentLaunchApproval = action("confirmExperimentLaunchApproval"),
-  onLaunchClicked = action("onLaunchClicked"),
-  onLaunchToPreviewClicked = action("onLaunchToPreviewClicked"),
-  ...props
-}: React.ComponentProps<typeof SubjectDraftStatusOperations>) => (
-  <SubjectDraftStatusOperations
-    {...{
-      rejectExperimentLaunch,
-      approveExperimentLaunch,
-      confirmExperimentLaunchApproval,
-      onLaunchClicked,
-      onLaunchToPreviewClicked,
-      ...props,
-    }}
-  />
-);
-
-storiesOf("pages/RequestReview/EXP-1055/DraftStatusOperations", module)
-  .addDecorator(withLinks)
-  .addDecorator((story) => <div className="p-5">{story()}</div>)
-  .add("review not requested", () => (
-    <SubjectDraftStatusOperationsWithActions />
+  .add("review pending in Remote Rettings, user can review", () => (
+    <SubjectEXP1143
+      {...{
+        reviewRequestEvent: mockChangelog("abc@mozilla.com"),
```

Comment:
Might not be worth changing now, but since `mockChangelog` has a default value, you could make a bunch of these calls parameter-less.